src/Chapter1.hs

```diff
@@ -542,7 +545,10 @@ value after "=" where the condition is true.
 Casual reminder about adding top-level type signatures for all functions :)
 -}
 
-mid x y z = error "mid: not implemented!"
+mid x y z 
+    | x < y = if y < z then y else z
```

Comment:
Here could be the case that `x > z`, and in this case, you would need to return `x` 🙂